fabric-rendering-fluids-v1/src/client/java/net/fabricmc/fabric/api/client/render/fluid/v1/FluidRenderHandlerRegistry.java

```diff
@@ -41,6 +43,16 @@ public interface FluidRenderHandlerRegistry {
 	 */
 	FluidRenderHandler get(Fluid fluid);
```

Comment:
Should this be marked with `@Nullable` as well?

Reply:
Done, thanks.